scripts/scil_evaluate_bundles_individual_measures.py

```diff
@@ -4,10 +4,10 @@
 """
 Evaluate basic measurements of bundles, all at once.
 All tractograms must be in the same space (aligned to one reference)
-
 The computed measures are:
 volume, volume_endpoints, streamlines_count, avg_length, std_length,
-min_length, max_length, mean_curvature
+min_length, max_length, span, curl, diameter, elongation, mean_curvature
+Note: volume is calculated differently in scilpy with density
```

Comment:
We determined this was not the case, right?

Also I would add a paragraph explaining what are the `set` stats and how they are computed. They are not just an average of the average, but they are weigthed by streamlines count. Explain why you think this make more sense and how you think we can interpret this.

Reply:
Also, maybe reference the paper I sent you from Frank Yeh (or the DSI studio website) for 'his' measures.
Look for `EPILOG` in other script to see how to cite publication in a script.